blog/_posts/2020-06-18-the-right-display-for-now.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: "the-right-display-for-now"
+date: 2020-06-18
+author: nick
+layout: blog
+title: "The Right Display for Now"
+image: "/assets/images/the-right-display-for-now/forest-park.jpg"
+tags:
+  - tilt
+  - ui
+---
+
+At Tilt, [Han and I](https://tilt.dev/about) talk a lot about the different
+modes developers are in and problems they're solving when using tools. Sometimes
+it's easier to talk about these modes when we relate them to signs in the real
+world.
+
+For example:
+
+- Discovery - what is it possible to do here? (see: the Forest Park sign above)
+- Awareness - what issues should I be aware of? (see: the estimated arrival time signs at train stations)
+- Recommendations - what should I do next? (see: exit signs, recommended routes)
+
+Tilt has always initialized your terminal with a terminal-based GUI. 
+
+We've found that the terminal-based GUI is good for awareness of all your
+different services. But if you're not actively swimming in the deep end of the
+Kubernetes ecosystem, the terminal GUI does a poor job communicating what you
```

Comment:
I would frame this differently. Something like this...(not final copy! Just spitballing in detail so you get a sense of what I mean...)

--------------------------------------------------

We'd previously thought of the split between Terminal GUI and Web as [awareness vs investigation](https://blog.tilt.dev/2019/04/09/designing-a-better-interface-for-microservices-development.html). But in the year since we launched the web version of Tilt, we've heard from users who actually see these as two competing interfaces. At a quick glance, they can seem equivalent, like it's a matter of preference which one you'd like to use. But in reality, we have invested far more in the web version. 

[Insert the great intro about Discovery / Awareness / Recommendations, underscoring how we're designing and making the web version with these in mind...and perhaps contrasting that with the fact our Terminal GUI is all but frozen in time. lol]

While we can imagine that the Terminal GUI will always remain valuable to some of our users, we realize that we need to communicate more clearly about how you get the best possible experience of Tilt.

In Tilt v0.15.0, we've made the role of the Terminal GUI more clear.

...

--------------------------------------------------

**More context on why I'm suggesting this framing**: I don't really even think our current terminal-based GUI is actually that good for awareness of your different services. While what I wrote in that old blog post about awareness vs investigation is good in theory, that workflow isn't actually reflected in the design of the TUI and Web Tilt UX. (Just think about how, say, the Apple Watch version of Music clearly differs from the desktop version. )

And I don't think there's anything inherent about the medium of TUI that would make it impossible to focus on discovery and recommendation. I can imagine super-focused TUI apps that do any of these things. But, that's just not a priority for us because realistically we haven't spent much time really digging into and understanding the use cases around CLI / TUI.

So, for me, this move de-emphasizing TUI is less about the limitations of what we can do in a given medium, and more about pragmatically consolidating our focus and really making sure our users are guided along a path where they'll have the best possible experience.

Reply:
I like that text a lot! I imported it and tweaked a bit